src/main.js

```diff
@@ -14,6 +14,7 @@ const ignoreTestFailures = core.getInput('ignore-test-failures') == 'true';
 const shouldCreateStatusCheck = core.getInput('create-status-check') == 'true';
 const shouldCreatePRComment = core.getInput('create-pr-comment') == 'true';
 const updateCommentIfOneExists = core.getInput('update-comment-if-one-exists') == 'true';
+const commentIdentifier = core.getInput('comment-identifier') == '';
```

Comment:
```suggestion
const commentIdentifier = core.getInput('comment-identifier');
```

If you were intending for this to be an empty string it would be more like:
 ```const commentIdentifier = core.getInput('comment-identifier') || '';```

Reply:
A default value as an empty string is what I went for here.